pkgs/development/python-modules/typed-argparse/default.nix

```diff
@@ -0,0 +1,32 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  setuptools,
+  typing-extensions,
+}:
+buildPythonPackage rec {
+  name = "typed-argparse";
+  version = "0.3.1";
+
+  pyproject = true;
+
+  src = fetchPypi {
+    pname = name;
+    inherit version;
+    hash = "sha256-OqxhyqUCBuCA0JoAw/5VK8TmQnOb6u+J9fjBExtdWv4=";
+  };
```

Comment:
Maybe fetch this from source? https://github.com/typed-argparse/typed-argparse
Upstream has a suite test

Reply:
Done! I've also bumped to the tip of their master branch, given (a) they aren't tagging releases and I don't want to have to do a commit-by-commit comparison to see what commit matches what they uploaded to Pypi, and (b) the latest commit adds support for Python 3.13, which I think we want.